meshtastic/remote_hardware.py

```
@@ -1,66 +1,164 @@
+
+        Raises
+        ------
+        MeshInterface.MeshInterfaceError
+            If no destination node ID is provided.
+        """
         if not nodeid:
-            our_exit(
-                r"Warning: Must use a destination node ID for this operation (use --dest \!xxxxxxxxx)"
+            raise _get_mesh_interface_error()(
+                "Must use a destination node ID for this operation."
             )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)